Probesys/bileto
# tec: Refactor the search forms

## Related issue(s)



https://github.com/Probesys/bileto/issues/551

## How to test manually



1. Use the ticket search forms
2. Verify that they work as expected

## Checklist
